transform/models/intermediate/clearinghouse/int_clearinghouse__five_minute_station_agg.sql

```diff
@@ -38,6 +38,24 @@ aggregated as (
         avg(occupancy) as occupancy -- Average of all the occupancy values
     from station_raw
     group by id, lane, sample_date, sample_timestamp_trunc
+),
+
+aggregated_speed as (
+    select
+        *,
+        --For speed I used the following formula to get speed: 
+        --SPEED = SUM(FLOW)/AVG(OCCUPANCY)/600 which resulted in 
+        --values from 0 to 5
+        --On 3/22/24 I updated the formula to use a vehicle effective length
+        --of 22 feet (16 ft vehicle + 6 ft detector zone) feet and using 
+        --a conversion to get miles per hour (5280 ft / mile and 12
+        --5-minute intervals in an hour).
+        case
+            when volume = 0 or occupancy = 0 then 0
+            when volume is null or occupancy is null then null
+            else (volume * 22) / occupancy * (1 / 5280) * 12
+        end as speed
```

Comment:
A few notes to make this logic a bit simpler and more efficient to calculate:

1. We don't need to check when `volume=0`, since the formula already evaluates to zero if volume is 0.
2. When `null` is involved in an arithmetic operation, the result is always `null`, so the second check isn't needed.
3. We can use the convenient `nullifzero` function to help with the `occupancy=0` check.

Putting those all together, we can replace this somewhat complicated case statement with a simpler formula that should also be faster:

```suggestion
        (volume * 22) / nullifzero(occupancy) * (1 / 5280) * 12 as speed
```